src/services/auth/reset.ts

```diff
@@ -1,37 +1,36 @@
-import { User } from '../../sequelize'
-import { resetEmail } from '../../helpers/nodemailer'
+import { User } from '../../sequelize';
+import { resetEmail } from '../../helpers/nodemailer';
 
-export default function reset(req, res, next) {
+export default function reset(req: any, res: any, next: any): void {
+  const { email } = req.body;
 
-  let { email } = req.body
+  User.findOne({ where: { email } })
+    .then((user) => {
+      if (!user) {
+        return res.status(400).json({
+          err: true,
+          msg: 'User with provided email does not exists',
+        });
+      }
 
-  User.findOne({where: { email }})
-  .then(user => {
-    if (user == null) {
-      return res.status(400).json({
-        err: true,
-        msg: 'User with provided email does not exists'
-      })
-    }
-
-    resetEmail(user.email, user.token)
-    .then(info => {
-      res.status(200).json({
-        err: false,
-        msg: 'Email with reset link sent successfully'
-      })
+      resetEmail(user.email, user.token)
+        .then((info) => {
+          res.status(200).json({
+            err: false,
+            msg: 'Email with reset link sent successfully',
+          });
+        })
+        .catch((err) => {
+          res.status(500).json({
+            err: true,
+            msg: 'There was an error with sending the reset link email',
+          });
+        });
     })
-    .catch(err => {
+    .catch((err) => {
```

Comment:
It's not required only if we omit the parentheses https://github.com/airbnb/javascript#arrows--one-arg-parens